### DISCUSSION TITLE
Rice Text Editor / WYSIWYG Editor

### DISCUSSION THREAD

**Discussion by mehedihasan95** [ant-design/ant-design#51735] [Ideas]
I want antd next update html rich text editor.

**Comment by petercat-assistant** [comment]
您希望在 Ant Design 的下一个更新中加入一个 HTML 富文本编辑器（WYSIWYG Editor）。目前，Ant Design 并没有内置的富文本编辑器组件，但您可以使用一些第三方库与 Ant Design 结合使用，例如 `react-quill` 或 `draft-js`。

如果您希望 Ant Design 官方集成一个富文本编辑器，您可以在 GitHub 上的讨论区或 issue 区提出这个功能请求，以便开发团队考虑在未来的版本中加入。

如果您有其他问题或需要进一步的帮助，请在评论中描述您的问题并 @petercat-assistant 以开始与我对话。
